tests/core/test_input_validation.py

```diff
@@ -767,13 +767,21 @@ def _check_is_sorted_params(arr):
     num_elements = np.prod(shape)
     arr_strict_ascending = np.arange(num_elements).reshape(shape)
 
+    # needed to support numpy <1.25
```

Comment:
```suggestion
    # needed to support numpy <1.24 and vtk 9.0.3
```
See:
https://github.com/pyvista/pyvista/blob/e84354e72acf6d69277954cdc0f16c739aab6897/.github/workflows/testing-and-deployment.yml#L186-L188